lib/api/web-element/factory.js

```diff
@@ -132,10 +132,18 @@ const createScopedWebElements = function(commandName, selector, parentElement, n
   const exported = {};
 
   Object.defineProperty(exported, 'then', {
+    value(onFulfilled, onRejected) {
+      return instance.then(onFulfilled, onRejected);
+    },
+    enumerable: true,
+    configurable: true
+  });
+
+  Object.defineProperty(exported, 'getElements', {
```

Comment:
Not necessarily, but I think it would be a nice to have, as I showed in the PR description. Like instead of `getValue()` in the example above, we could use `isVisible()` to get all the elements which are visible and perform operations on them accordingly.

Reply:
Should I remove this? Or we can keep this method?

Reply:
let's remove it to avoid confusion. We can revisit it later if needed.